src/main/scala/org/opennetworkinsight/dns/model/DNSScoreFunction.scala

```diff
@@ -0,0 +1,57 @@
+package org.opennetworkinsight.dns.model
+
+import org.apache.spark.broadcast.Broadcast
+import org.opennetworkinsight.SuspiciousConnectsScoreFunction
+import org.opennetworkinsight.dns.DNSWordCreation
+
+
+/**
+  * Estimate the probabilities of network events using a [[DNSSuspiciousConnectsModel]]
+  *
+  * @param frameLengthCuts
+  * @param timeCuts
+  * @param subdomainLengthCuts
+  * @param entropyCuts
+  * @param numberPeriodsCuts
+  * @param topicCount
+  * @param ipToTopicMixBC
+  * @param wordToPerTopicProbBC
+  * @param topDomainsBC
+  */
+class DNSScoreFunction(frameLengthCuts: Array[Double],
+                       timeCuts: Array[Double],
```

Comment:
I don't like passing in almost all the state from the DNSSuspiciousConnectsModel,  but I don't want to have to Serialize that thing....